srcpkgs/pcsc-cyberjack/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'pcsc-cyberjack'
+pkgname=pcsc-cyberjack
+version=3.99.5
+revision=1
+_versext=SP15
+wrksrc=pcsc-cyberjack-${version}final.${_versext}
+build_style=gnu-configure
+configure_args="--enable-udev"
+hostmakedepends="pkg-config"
+makedepends="pcsclite-devel eudev-libudev-devel libusb-compat-devel"
+depends="pcsc-ccid libusb"
```

Comment:
`libusb` shouldn't be necessary, it's detected: `SONAME: libusb-1.0.so.0 <-> libusb>=1.0.0_1`

Reply:
I've removed the libusb dependency and it build without a problem. Thanks for your hint. :+1: